application/logger.py

```
@@ -0,0 +1,128 @@
+        if not self.backup_count:
+            return
+
+        log_files = sorted(
+            self.log_dir.glob(f"{self.filename_prefix}_*.clp.zst"),
+            key=os.path.getmtime,
+        )
+        if len(log_files) > self.backup_count:
+            for old_file in log_files[:-self.backup_count]:
+                try:
+                    old_file.unlink()
+                except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)